config/pretty-routes.php

```diff
@@ -69,6 +69,32 @@
 
     'color_scheme' => 'auto',
 
+    /*
+     * Apply color theme when idle time or active.
+     *
+     * If your browser theme following any specific condition (os, time etc.) helps to auto theme switch.
+     *
+     * 1000 ms = 1 seconds
+     *
+     * This option depends to "color_scheme" and works only with "auto"
+     *
+     * By default, 1000 and accecpts only integer values.
+     */
+
+    'color_scheme_idle_time' => 1000,
```

Comment:
No, when return back to tab and if the idle delay timeout then triggers to switch.

My scenario:

1. Go to another tab.
2. Return back after idle time.
3. Automatically theme switching is triggered.

I think idle time good idea because I am using only checking routes. Return back interval depends to developer. 
If you set as 0 it is going to switches immediately when return back.

Reply:
Hmmm... What if I don't want to switch the theme automatically?

The "auto" option now responds to the selected theme in the browser. If the browser changes to dark or light, the site will "see" this change.

Reply:
Yes but current theme option works only startup. `color_scheme_idle_time` follows `color_scheme` option. If already user set as `auto` this option provides auto switch. It is plus.

Reply:
Okay. Let's see how it works :)